.gitignore

```diff
@@ -31,3 +31,5 @@ cmd/newrelic-infra/versioninfo.json
 /vendor
 !/vendor/modules.txt
 .envrc
+.floo
+.flooignore
```

Comment:
as these gitignore entries are custom setup centric I think $HOME is a better place for these ones, don't you?

Reply:
I was trying out https://floobits.com/ with @brushknight for pair programming so we both had this code checked out. Added it to make the setup up easier for both of us. It definitely makes sense to move it to `$HOME` for now but might be worth looking at if we continue to use floobits.